src/components/Cart/CartItemDisplay.js

```diff
@@ -0,0 +1,215 @@
+import { React, useState, useEffect } from 'react';
+import Airtable from 'airtable';
+import {
+  Grid, Typography, IconButton, makeStyles, ButtonBase,
+} from '@material-ui/core';
+import { Remove, Add } from '@material-ui/icons';
+import Delete from '@material-ui/icons/DeleteOutlineOutlined';
+import PropTypes from 'prop-types';
+import './Cart.css';
+import { RemoveConfirmationDialog, MaxAvailableDialog } from './CartDialogs';
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+const useStyles = makeStyles({
+  listingImage: {
+    height: '83px',
```

Comment:
That's a great idea! We should definitely look into breakpoints. For now though since we have less time, we can write a little TODO message to explore breakpoints and media queries, and circle back on this